AzureStack/2015-04-01/swagger/GalleryAdmin.json

```diff
@@ -0,0 +1,502 @@
+        ],
+        "description": "Lists gallery registrations",
+        "operationId": "ProviderGalleryRegistrations_List",
+        "consumes": [],
+        "produces": [
+          "application/json",
+          "text/json"
+        ],
+        "parameters": [
+          {
+            "$ref": "#/parameters/ApiVersionParameter"
+          },
+          {
+            "$ref": "#/parameters/SubscriptionIdParameter"
+          },
+          {
+            "$ref": "#/parameters/resourceGroupNameParameter"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "OK",
+            "schema": {
+              "type": "array",
+              "items": {
+                "$ref": "#/definitions/ResourceDefinition"
+              }
+            }
+          }
+        }
+      }
+    }
+  },
+  "definitions": {
+     "GalleryItemUriPayload": {
+      "description": "Gallery item URI, Points to the http or https uri where the galery package is located, typically a blob uri",
+      "type": "object",
+      "properties": {
+        "GalleryItemUri": {
+          "description": "Gets or sets the gallery item URI.",
+          "type": "string"
+        }
+      }
+    },
+    "ResourcePlan": {
+      "type": "object",
+      "description": "The reosurce plan object",
+      "properties": {
+        "Name": {
+          "type": "string",
+          "description": "The name of the plan",
+          "readOnly": true
+        },
+        "Product": {
+          "type": "string",
+          "description": "The product code of the plan",
+          "readOnly": true
+        },
+        "PromotionCode": {
+          "type": "string",
+          "description": "The promotion code of the plan",
+          "readOnly": true
+        },
+        "Publisher": {
+          "type": "string",
+          "description": "The publisher of the plan",
+          "readOnly": true
+        }
+      }
+    },
+    "ResourceSku": {
+      "type": "object",
+      "description": "The resource sku object",
+      "properties": {
+        "Name": {
+          "type": "string",
+          "description": "The name of the sku",
+          "readOnly": true
+        },
+        "Tier": {
+          "type": "string",
+          "description": "The tier of the sku",
+          "readOnly": true
+        },
+        "Size": {
+          "type": "string",
+          "description": "The size of the sku",
+          "readOnly": true
+        },
+        "Family": {
+          "type": "string",
+          "description": "The family of the sku",
+          "readOnly": true
+        },
+        "Capacity": {
+          "format": "int32",
+          "type": "integer",
+          "description": "The capacity of the sku",
+          "readOnly": true
+        }
+      }
+    },
+    "ResourceDefinition": {
+      "type": "object",
+      "description": "The resource definition object of the azure resource manager",
+      "properties": {
+        "Id": {
```

Comment:
Looks like you are looking at the WebAPI models which are in C#.  That is why all the model properties are in PascalCase. Swagger is a representation of the wire format. On the wire everything is in camelCase. Open Fiddler and see what is happening on the wire and then name things accordingly.